### ISSUE TITLE
Regarding NER finetuning

### ISSUE THREAD

**Issue by malduwais** [google-research/xtreme#103]
Dear Authors, 

I am trying to re-produce the NER results using WikiANN. However, I didn't get the same results even I am using the same hyperparameter for XLM-R and mBERT models. 

Is there something I need to set other than the hyperparameter?

I am using Trainer method as follow:

args = TrainingArguments(
    model_path,
    overwrite_output_dir=True,
    num_train_epochs=10,
    load_best_model_at_end=True,
    evaluation_strategy="epoch",
    save_strategy="epoch",
    report_to="wandb",
    seed=seed,
    data_seed=seed,
    per_device_train_batch_size=32,
    per_device_eval_batch_size=32,
    learning_rate=2e-5,
    weight_decay=0.01,
    warmup_ratio=0.1,
)

Thank you,
